docs/src/components/HeaderMenu.vue

```diff
@@ -107,6 +107,16 @@ div
           q-icon(:name="fabJsfiddle")
         q-item-section jsFiddle
 
+      q-item(clickable, href="https://stackblitz.com/fork/quasarframework", target="_blank", rel="noopener")
+        q-item-section.text-black(avatar)
+          q-icon(:name="fasBolt")
```

Comment:
Maybe we could make these blue? `color="blue"`

Reply:
Nice suggestion, I went with `light-blue-14`.